### ISSUE TITLE
MYSQL on IPv6 docker network

### ISSUE THREAD

**Issue by moutasem1989** [haiwen/seafile-docker#396]
Creating an IPv6 Docker network, makes MYSQL unreachable to Seafile.
I assume that in bootstrap.py, the User Host should be `'%'` instead of `'MYSQL_USER_HOST': '%.%.%.%',`. Otherwise MYSQL would throw an error „access denied“ when trying over IPv6.

**Comment by herver** [comment]
Indeed, I've been pulling my hair off for the past couple of hours only to realize this, 

I think someone already fixed this exact problem in #288

**Comment by SkywalkerSpace** [comment]
Thank you for your feedback, we fixed this issue in this commit <https://github.com/haiwen/seafile-docker/commit/2ac31237e565d4838d4ed05a9f055498135ef50a>
